test/test-loginController.js

```diff
@@ -111,36 +104,77 @@ describe('loginController', function() {
     });
 
     describe('postLogin', () => {
+      let insertTokenStub;
+      let getPsychologistByEmailStub;
+      let sendMailStub;
+
+      beforeEach(async function() {
+        insertTokenStub = sinon.stub(dbLoginToken, 'insert')
+          .returns(Promise.resolve());
+
+        getPsychologistByEmailStub = sinon.stub(dbPsychologists, 'getPsychologistByEmail')
+          .returns(Promise.resolve({
+            email: email
+          }));
+
+        sendMailStub = sinon.stub(emailUtils, 'sendMail')
+          .returns(Promise.resolve());
+      })
+
+      afterEach(function(done) {
+        insertTokenStub.restore();
+        getPsychologistByEmailStub.restore();
+        sendMailStub.restore();
+        done();
+      })
+
       it('send a login email', function(done) {
         chai.request(app)
```

Comment:
On peut rajouter un test "should refuse login if csrf token is invalid" ?

Reply:
tu lis dans mes pensées

Reply:
:heavy_check_mark:

Reply:
https://github.com/betagouv/sante-psy/pull/63/commits/296d593ad8229c7cb1ebab07e367d49a429353c8#diff-f640ac19eb8202e21715386b7e4ae1918a86abba9de3196a6acb9614b4813906R170